File: Baekjoon/Baekjoon01929.cpp
// Baekjoon01929.cpp
#include <iostream>

#include <vector>

using namespace std;

int main(void) {
    int n, m;
    cin >> n >> m;

    vector<bool> isPrime(m + 1, true);
    isPrime[0] = isPrime[1] = false;

    for (int i = 2; i * i <= m; i++) {
        if (isPrime[i])
            for (int j = i * i; j <= m; j += i)
                isPrime[j] = false;
    }

    for (int i = n; i <= m; i++)
        if (isPrime[i])
            cout << i << '\n';

    return 0;
}

File: Baekjoon/Baekjoon01929.py
# Baekjoon01929.py
n, m = map(int, input().split())

is_prime = [True] * (m + 2)

is_prime[0] = is_prime[1] = False
for i in range(2, m + 1):
    if is_prime[i]:
        if n <= i:
            print(i)

        for j in range(i * i, m + 1, i):
            is_prime[j] = False
